script.js

```diff
@@ -0,0 +1,180 @@
+  } else {
+    for (let i = 0; i < hide.length; i += 1) {
+      hide[i].classList.add('hidden');
+    }
+    document.querySelector('.more-btn .arrow .up').classList.add('hidden');
+    document.querySelector('.more-btn .arrow #down').classList.remove('hidden');
+    document.querySelector('.more-btn .text').textContent = 'MORE';
+  }
+};
+
+document
+  .querySelector('.more-btn')
+  .addEventListener('click', moreElements);
+
+//  The menu
+// Open the menu
+
+const openMenu = function () {
+  document.querySelector('.menu.mobile').classList.remove('hidden');
+  document.querySelector('.buyTicket').style.color = 'white';
+  document.querySelector('html').style.overflowY = 'hidden';
+  document.querySelector('body').style.overflowY = 'hidden';
+  if (window.screen.width >= 768) {
+    document.querySelector('.buyTicket').style.color = '#ec5242';
+  }
+};
+// Close the menu
+
+const closeMenu = function () {
+  document.querySelector('.menu.mobile').classList.add('hidden');
+  document.querySelector('body').style.overflowY = 'initial';
+  document.querySelector('html').style.overflowY = 'initial';
+};
+
+const menuOpenerCloser = function () {
+  if (document.getElementsByClassName('menu')[0].classList.contains('hidden')) {
+    openMenu();
+  } else {
+    closeMenu();
+  }
+};
+
+document.getElementsByClassName('menu_')[0].addEventListener('click', menuOpenerCloser);
+
+document
+  .getElementsByClassName('cancel')[0]
+  .addEventListener('click', menuOpenerCloser);
+
+// Show Home Page
+
+const showHome = function () {
+  document.querySelector('.text-container').classList.remove('hidden');
+  document.querySelector('.text-containerAbout-page').classList.add('hidden');
+  document.querySelector('.container p.display-1').style.textAlign = '';
+  document.querySelector('.container h1').style.textAlign = '';
+  document.getElementsByClassName('mainProgram')[0].classList.remove('hidden');
+  document
+    .getElementsByClassName('featuredSpeakers')[0]
+    .classList.remove('hidden');
+  document.getElementsByClassName('dateAbout')[0].classList.add('hidden');
+  document.getElementsByClassName('date')[0].classList.remove('hidden');
+  document.getElementsByClassName('about-page')[0].classList.add('hidden');
+  document.getElementsByClassName('Partner')[0].classList.add('hidden');
+  document.querySelector('.container h1').style.fontSize = '32px';
+  document.querySelector('.text-container').classList.remove('hidden');
+  document.querySelector('.text-containerAbout-page').classList.add('hidden');
+  document.querySelector('.PartnerContainer').classList.add('hidden');
+  document.querySelector('.container').style.marginLeft = '0';
+  document.querySelector('.home-page h1').style.color = '#ec534271';
+  document.querySelector('.container').style.alignItems = 'none';
+  document.querySelector('body').style.overflowY = 'initial';
+  document.querySelector('html').style.overflowY = 'initial';
```

Comment:
### [OPTIONAL]
- [ ] TO make this code DRY and KISS, I think we would improve it by putting all the styles in a CSS file, the javascript file should not contain styles, this will make our javascript clear and maintainable.